client/src/GamemasterInGame/PlayerListComponents/index.tsx

```diff
@@ -14,4 +15,6 @@ export const PlayerList = {
   MenuItem: PlayerMenuItem,
   NoteInputModal: PlayerNoteInput,
   PlayerMessage: PlayerMessageFlow,
+  // TODO: Route PlayerControls through here. SpectatorControls to route elsewhere in another PR.
+  PlayerControls: PlayerControls,
```

Comment:
We've been using "Actions"

Reply:
Also, this shouldn't even go here, since this is in the GM view, adn you are changing the player view.